examples/simple/main.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2019 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// This example script is the sample from the OpenTelemetry Go "Getting Started"
+// guide, with the text-based exporter replaced with the New Relic OpenTelemetry
+// Exporter.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/newrelic/newrelic-telemetry-sdk-go/telemetry"
+	"github.com/newrelic/opentelemetry-exporter-go/newrelic"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/baggage"
+	"go.opentelemetry.io/otel/label"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/sdk/metric/controller/push"
+	"go.opentelemetry.io/otel/sdk/metric/processor/basic"
+	"go.opentelemetry.io/otel/sdk/metric/selector/simple"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func main() {
+
+	// Create a New Relic OpenTelemetry Exporter
+	apiKey, ok := os.LookupEnv("NEW_RELIC_API_KEY")
+	if !ok {
+		fmt.Println("missing NEW_RELIC_API_KEY required for New Relic OpenTelemetry Exporter")
+	}
+
+	exporter, err := newrelic.NewExporter(
+		"Sample OpenTelemetry Service",
+		apiKey,
+		telemetry.ConfigBasicErrorLogger(os.Stderr),
+		telemetry.ConfigBasicDebugLogger(os.Stderr),
+		telemetry.ConfigBasicAuditLogger(os.Stderr),
+	)
+	if err != nil {
+		fmt.Printf("failed to instantiate New Relic OpenTelemetry exporter: %v\n", err)
+	}
+
+	ctx := context.Background()
+	defer exporter.Shutdown(ctx)
+
+	// Create a tracer provider
+	bsp := sdktrace.NewBatchSpanProcessor(exporter)
+	tp := sdktrace.NewTracerProvider(sdktrace.WithSpanProcessor(bsp))
+	defer func() { _ = tp.Shutdown(ctx) }()
+
+	// Create a meter provider
+	pusher := push.New(
+		basic.New(
+			simple.NewWithExactDistribution(),
+			exporter,
+		),
+		exporter,
+	)
+	pusher.Start()
+	defer pusher.Stop()
+
+	// Set global options
+	otel.SetTracerProvider(tp)
+	otel.SetMeterProvider(pusher.MeterProvider())
+	propagator := propagation.NewCompositeTextMapPropagator(propagation.Baggage{}, propagation.TraceContext{})
+	otel.SetTextMapPropagator(propagator)
+
+	// Sample metric instruments
+	fooKey := label.Key("ex.com/foo")
+	barKey := label.Key("ex.com/bar")
+	lemonsKey := label.Key("ex.com/lemons")
+	anotherKey := label.Key("ex.com/another")
```

Comment:
Would it be helpful to change the prefix to `simple`? It makes it obvious that the data came from the simple example.

Relatedly, what is the recommended practice in the OTel community for naming metrics? Do these metric names follow those practices?

Reply:
I'm not familiar with metric naming practices in OpenTelemetry, but these metric names come straight from the [Getting Started](https://github.com/krnowak/opentelemetry.io/blob/1d498b5c2d578cd1932cb822ef482fb3ef861c21/content/en/docs/go/getting-started.md#creating-metric-instruments) guide written by the OpenTelementry Go folks.

Changing it would add to the confusion.